go/samples/menu/s02.go

```diff
@@ -51,12 +51,10 @@ func setup02(_ context.Context, m ai.Model) error {
 
 		 Question:
 		 {{question}} ?`,
-		dotprompt.Config{
-			Model:        m,
-			InputSchema:  menuQuestionInputSchema,
-			OutputFormat: ai.OutputFormatText,
-			Tools:        []ai.Tool{menuTool},
-		},
+		dotprompt.WithDefaultModel(m),
+		dotprompt.WithInputType(menuQuestionInput{}),
+		dotprompt.WithOutputFormat(ai.OutputFormatText),
```

Comment:
Let's omit this from examples and automatically set output format to text if unset.